src/containers/Tenants/Tenants.js

```diff
@@ -154,7 +147,6 @@ class Tenants extends React.Component {
                                     name: value,
                                     backend,
                                     [TenantTabsGroups.info]: initialTenantInfoTab,
-                                    [TenantTabsGroups.general]: initialTenantGeneralTab,
```

Comment:
If undefined, it's taken from settings in tenant page. There is no need to define it explicitly